Zone.lua

```diff
@@ -0,0 +1,173 @@
+-----------------------------------
```

Comment:
You have this file in the root of the repository instead of in `scripts/zones/RuLude_Gardens/`